CHANGELOG.md

```diff
@@ -21,6 +21,8 @@ Our versioning strategy is as follows:
 
 * `[sitecore-jss-nextjs]` Enforce CORS policy that matches Sitecore Pages domains for editing middleware API endpoints ([#1798](https://github.com/Sitecore/jss/pull/1798)[#1801](https://github.com/Sitecore/jss/pull/1801))
 
+* `[nextjs-xmcloud]` DictionaryService will now use a `site` GraphQL query instead of `search` one out of the box to improve performance ([#1804](https://github.com/Sitecore/jss/pull/1804))
```

Comment:
We need to write that DictionaryService **CAN** use _site_ query, because by default it still uses _search_ query. But it's enabled in nextjs-xmcloud add-on out of the box